.gitlab-ci.yml

```diff
@@ -107,6 +108,91 @@ workflow:
       vault:                       cicd/gitlab/$CI_PROJECT_PATH/GITHUB_SSH_PRIV_KEY@kv
       file:                        false
 
+#### stage:                        lint
+#
+# Note: For all of these lints we `allow_failure` so that the rest of the build can
+# continue running despite them not passing. Merging is still disallowed since (most) of
```

Comment:
Just as an FYI: I checked and it's only `spellcheck` which is currently not required for merging.